src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import fakeBookings from "./fakeBookings.js";
+
+import Tbody from "./Tbody.js";
+// console.log(fakeBookings);
```

Comment:
Please don't send commented code when submit pull requests.

Reply:
thank you, i did not know will work on that